plugins/LadspaEffect/caps/dsp/Eq.h

```diff
@@ -26,9 +26,9 @@
 	02111-1307, USA or point your web browser to http://www.gnu.org.
 */
 
+
 #ifndef _DSP_EQ_H_
 #define _DSP_EQ_H_
-
```

Comment:
@Rossmaxx can you explain the reasoning behind this minor formatting change?  Is it intended to be upstreamed?